edx_proctoring/templates/proctoring/seq_timed_exam_footer.html

```diff
@@ -0,0 +1,11 @@
+{% load i18n %}
+<div class="footer-sequence">
+  <h4> {% trans "What if i need more time " %}? </h4>
```

Comment:
@chrisndodge Can you confirm when this footer appears? This information would be most helpful only if it appears *before* a learner takes the timed exam.